counsel.el

```diff
@@ -269,10 +268,13 @@ Update the minibuffer with the amount of lines collected every
              (push (symbol-name x) symbol-names))))
       (setq symbol-names (all-completions str obarray pred)))
     (ivy-read "Symbol name: " symbol-names
+              :caller 'counsel-el
               :predicate pred
               :initial-input str
               :action #'ivy-completion-in-region-action)))
 
+(add-to-list 'ivy-height-alist '(counsel-el . 7))
```

Comment:
The docstring of `add-to-list` says

    This is handy to add some elements to configuration variables,
    but please do not abuse it in Elisp code, where you are usually
    better off using ‘push’ or ‘cl-pushnew’.

Reply:
it's precisely configuration stuff.

Reply:
While you're not wrong, and it shouldn't make a difference here, "configuration" generally refers to user customisations, i.e. init file code. In particular, `add-to-list` can misbehave under lexical scope/extent.